client/src/components/JurisdictionAdmin/generateSheets.ts

```diff
@@ -795,3 +797,128 @@ export const downloadTallyEntryLoginLinkPrintout = async (
   )
   return doc.output() // returned for test snapshots
 }
+
+const AUDIT_BOARD_SIGN_MAX_CONTEST_NAME_LENGTH = 40
+const AUDIT_BOARD_SIGN_MAX_SURNAME_LENGTH = 20
+const AUDIT_BOARD_SIGN_TITLE_LENGTH_XL_FONT_CUTOFF = 12
+const AUDIT_BOARD_SIGN_TEXT_COLOR = Colors.GRAY3
```

Comment:
I think this could be slightly darker - it's a little hard to read on screen. Though if it's better printed, I defer to your judgement

Reply:
I was trying to mimic what Ginny had in her doc, but agreed that slightly darker is better. Shifting to `Gray2`